internal/provider/resource_topic.go

```diff
@@ -46,17 +46,27 @@ func (t topicResourceType) GetSchema(ctx context.Context) (tfsdk.Schema, diag.Di
 				Required:            true,
 				Type:                types.StringType,
 				Validators: []tfsdk.AttributeValidator{
-					validation.Compare(validation.OneOf, []string{"JSON", "Binary", "String", "Xml"}),
+					validation.Compare(validation.OneOf, []string{"AVRO", "JSON", "Binary", "String", "Xml"}),
```

Comment:
please add key_type attribute level validations here to make tf plan work: "if key_type is AVRO, key_schema can't be empty". Please see my other comment for more details.